apps/web/lib/team/[slug]/[type]/getServerSideProps.tsx

```diff
@@ -86,6 +86,7 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
           length: true,
           hidden: true,
           hosts: {
+            take: 3,
```

Comment:
`hosts` are only used for OG image creation. 3 should be enough because names get cut anyways